app/modules/node-app-service/main.tf

```diff
@@ -97,6 +97,43 @@ resource "azurerm_linux_web_app_slot" "staging" {
   }
 }
 
+resource "azurerm_app_service_certificate" "custom_hostname" {
+  count = var.custom_hostname_enabled ? 1 : 0
+
+  name                = var.custom_hostname
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  key_vault_secret_id = var.custom_hostname_certificate_secret_id
+
+  tags = var.tags
+}
+
+resource "azurerm_app_service_custom_hostname_binding" "custom_hostname" {
+  count = var.custom_hostname_enabled ? 1 : 0
+
+  hostname            = var.custom_hostname
+  app_service_name    = azurerm_linux_web_app.web_app.name
+  resource_group_name = var.resource_group_name
+  # thumbprint          = azurerm_app_service_certificate.custom_hostname[0].thumbprint
```

Comment:
Can we remove this if we're not using? :)

Reply:
Comments removed in subsequent commits.